### ISSUE TITLE
font-family: "object-fit:cover";

### ISSUE THREAD

**Issue by bosonno** [mastodon/mastodon#22546]
### Steps to reproduce the problem

Incorrect font family for alt text when avatar image is missing.

### Expected behaviour

Correct font family should have been used.

### Actual behaviour

Incorrect font family used.

### Detailed description

.account__avatar img {
    display: block;
    width: 100%;
    height: 100%;
    -o-object-fit: cover;
    font-family: "object-fit:cover";
    object-fit: cover;
}

### Specifications

Mastodon 4.0.2

**Comment by ClearlyClaire** [comment]
IIRC this is on purpose to provide the `object-fit: cover` functionality through a polyfill. That being said, this is likely not needed anymore.

**Comment by ClearlyClaire** [comment]
It was added in #4182 and support for `object-fit` is now near-universal: https://caniuse.com/object-fit

So we may want to drop the polyfill.

**Comment by bosonno** [comment]
IIRC there's no font-family named: "object-fit:cover"
https://www.dafont.com/search.php?q=object-fit%3A+cover

**Comment by ClearlyClaire** [comment]
Of course, it's just used to store the value of the `object-fit` property somewhere where Edge can read it so that the polyfill can do its work.

**Comment by bosonno** [comment]
> IIRC there's no font-family named: "object-fit:cover" https://www.dafont.com/search.php?q=object-fit%3A+cover

yet

**Comment by shuuji3** [comment]
Thanks @ClearlyClaire, I found the same issue and investigated by myself but couldn't find the cause. That past PR you showed was beneficial! 🙂 

I created a PR to revert the old polyfill #22693.
